File: python/factorial.py
def factorial(num):
    if num == 0:
        return 1
    elif num < 0:
        return False
    else:
        my_num=num
        answer=1
        while my_num>0:
            answer=answer*my_num
            my_num-=1
        # print(answer)
        return answer

File: js/factorial.js
exports.factorial = function(num) {
    if(num===0){
        return 1
    }
    else if(num <0){
        return false
    }
    else{
        let answer=1
        let myNum=num
        while(myNum>0){
            answer=answer*myNum
            myNum--
        }
        console.log(answer)
        return answer
    }
};
